notebooks/vertex_matching_engine/solutions/matching_engine_ann.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<p>Maybe add a line explaining the effect of creating an endpoint in a particular VPC network using the <code>--network</code> argument. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>